io.dbeaver.DBeaverCommunity.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
```

Comment:
`dbeaver-ce-5.2.3-linux.gtk.x86_64.tar.gz` already have a `.desktop` file. You can reuse it and tweak it to your needs using `desktop-file-edit` – [here](https://github.com/flathub/com.wire.WireDesktop/blob/master/com.wire.WireDesktop.json#L74) is an example.

Moreover, it has pretty much the same issues/inconsistencies that reviewers are pointing out here, so if you have time you can send a pull request to `DBeaver` to fix those altogether.

Reply:
Is this preferred solution to including its own desktop file until they fix their desktop file?
I looked into their repo, since I'm not familiar with maven or how do they even produce desktop file this might have to wait.

Reply:
It's definitely less work for you, except that you would need to change some lines for now — which is pretty straightforward. As of right now, you two have two things; if you reuse that file, you would be closer to what they are delivering plus the changes you would need to do for this Flatpak.

I don't consider this a big deal anyway, but my preference is to go first for what's already done and tweak it 😉

Reply:
edited desktop file with desktop-file-edit, it doesn't look nice :)